Make aggregation functions consistent across IOGroups

avilcheslopez opened the pull request:
Update aggregation functions in CNL, LevelZero, and NVML IOGroups to
match how aggregation functions were defined for MSRs (for consistency
across all IOGroups).

Signed-off-by: Alejandro Vilches <alejandro.vilches@intel.com>

- Relates to #2505 
- Fixes #2546

avilcheslopez committed:
Make aggregation functions consistent across IOGroups

Update aggregation functions in CNL, LevelZero, and NVML IOGroups to
match how aggregation functions were defined for MSRs (for consistency
across all IOGroups).

Signed-off-by: Alejandro Vilches <alejandro.vilches@intel.com>

avilcheslopez committed:
Update aggregation functions in SSTIOGroup for consistency

Signed-off-by: Alejandro Vilches <alejandro.vilches@intel.com>

dannosliwcd commented on service/docs/source/geopm_pio_sst.7.rst:
```diff
@@ -136,7 +136,7 @@ Configuration
     Returns the CPU's assigned CLOS.
 
     * **Aggregation**: select_first
-    * **Domain**: package
```
Thanks for fixing that!

dannosliwcd reviewed (changes requested):
The rst docs for these ones don't get autogenerated like the MSR ones. So the changes to agg functions need to be manually applied.

avilcheslopez committed:
Update IOGroup man pages

Update IOGroup man pages to reflect changes in aggregation function for
some signals/controls.

Signed-off-by: Alejandro Vilches <alejandro.vilches@intel.com>

cmcantalupo reviewed (approved):
LGTM
